pkg/cluster/spec/validate.go

```diff
@@ -222,23 +223,26 @@ func CheckClusterPortConflict(clusterList map[string]Metadata, clusterName strin
 			blackboxExporterPort := mOpt.BlackboxExporterPort
 			for _, port := range inst.UsedPorts() {
 				existingEntries = append(existingEntries, Entry{
-					clusterName: name,
-					instance:    inst,
-					port:        port,
+					clusterName:   name,
+					componentName: inst.ComponentName(),
+					host:          inst.GetHost(),
```

Comment:
Um, I didn't quite get the point of adding a new field, we could use `inst.GetHost()` to get the value anytime.

Reply:
I try to make the message contains in `Entry` is clear and eliminate the ambiguous. If there exist fields `port`, `componentName` along with `inst`, should I get the port from `inst.GetPort()` or `inst.ComponentName()`?
However, keeping `inst` instead of adding `host` is more flexible.

I can keep `inst` instead of adding `host` if you insist.

Reply:
Yes I would still suggest using `inst` as it carries more information.